dev-python/backports-shutil_which/backports-shutil_which-3.5.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+MY_PN="backports.shutil_which"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Backport of shutil.which from Python 3.3"
+HOMEPAGE="https://pypi.python.org/pypi/backports.shutil_which/ https://github.com/minrk/backports.shutil_which"
+SRC_URI="mirror://pypi/${PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+SLOT="0"
```

Comment:
It's recommended (but not required) to follow the common variable ordering, which has ``LICENSE`` first followed by ``SLOT``.